src/dashboard/apigateway/apigateway/apps/release/views.py

```diff
@@ -149,7 +149,14 @@ def list(self, request, *args, **kwargs):
             fuzzy=True,
         )
         page = self.paginate_queryset(queryset)
-
+        # 查询发布事件
+        publish_ids = [release_history.id for release_history in page]
+        publish_last_event = PublishEvent.objects.get_publish_events_by_publish_ids(publish_ids)
```

Comment:
publish_last_event to publish_last_events

Reply:
done